scripts/clean_up.py

```diff
@@ -0,0 +1,112 @@
+import os
+import glob
+import shutil
+import tarfile
+import argparse
+
+from qcore import utils
+
+SUBMISSION_DIR_NAME = 'submission_temp'
+SUBMISSION_SUBDIR_NAME = 'slurm_and_logs'
+SUBMISSION_TAR = 'submission.tar'
+SUBMISSION_FILES = ['flist_LF', 'slurm_header.cfg', 'machine_env.sh', 'submit.sh', '*.template']
+SUBMISSION_SL_LOGS = ['*.sl', '*.err', '*.out']
+
+LF_DIR_NAME = 'LF_temp'
+LF_SUB_DIR_NAME = 'OutBin'
+LF_TAR = 'LF.tar'
+LF_FILES = ['Rlog', 'Restart', 'SlipOut']
+
+
+def make_tar(source_dir, out_dir):
+    """params: source_dir:source dir of all files to tar
+               out_dir: output dir for the tar.gz
+    """
+    try:
+        print("start making tar")
+        with tarfile.open(out_dir,'w:gz') as tar:
+            tar.add(source_dir,arcname=os.path.basename(source_dir))
+            print("finished making csv tar")
+    except Exception as e:
+        print(e)
+        print("cant'make tar")
+
+
+def move_files(sim_dir, dest_dir, file_list):
```

Comment:
Make docstring slightly clearer? Maybe something like "Moves all files that match any of the specified file patterns to destination directory"

Rename file_list to file_patterns, since its a list of patterns and not actual file names?